modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/ToolsEndpoint.java

```diff
@@ -649,24 +640,24 @@ public Response editVideo(@PathParam("mediapackageid") final String mediaPackage
       }
 
       // Update default thumbnail (if used) since position may change due to cutting
-      MediaPackageElement distributedThumbnail = null;
-      if (editingInfo.getDefaultThumbnailPosition().isPresent()) {
-        try {
-          final ThumbnailImpl thumbnailImpl = newThumbnailImpl();
-          final Optional<ThumbnailImpl.Thumbnail> optThumbnail = thumbnailImpl
-            .getThumbnail(mediaPackage, urlSigningService, expireSeconds);
-          if (optThumbnail.isPresent() && optThumbnail.get().getType().equals(ThumbnailImpl.ThumbnailSource.DEFAULT)) {
-            distributedThumbnail = thumbnailImpl
-              .chooseDefaultThumbnail(mediaPackage, editingInfo.getDefaultThumbnailPosition().getAsDouble());
-          }
-        } catch (UrlSigningException | URISyntaxException e) {
-          logger.error("Error while trying to serialize the thumbnail url because: {}", getStackTrace(e));
-          return R.serverError();
-        } catch (IOException | EncoderException | PublicationException | UnknownFileTypeException | MediaPackageException e) {
-          logger.error("Error while updating default thumbnail because: {}", getStackTrace(e));
-          return R.serverError();
-        }
-      }
+//      MediaPackageElement distributedThumbnail = null;
```

Comment:
@staubesv When I made the tests, even if you didn't touch the interactive thumbnail generation, if  you hit save and continue (When you made a cut in the video), that calls the composer to create the thumbnail as a default configuration (If you didn't upload any thumbnail or created one using the interactive method).

Nevertheless this is an emergency fix. I will work in a next PR to solve better and restore this feature.

Reply:
Agreed, this code has to go, as it can otherwise cause extremely slow responses to "Save and Continue" (POST to editor.json). This operation should happen in a workflow anyway, so it's unclear why it had to be included here and run before you get a response from the POST to editor.json.

Reply:
For this kind of job, workflows are not a valid option as workflows are a) far(!) too slow and b) appear in the history of processing. Here, we don't actually process the event. We just needs a preview image used by the editor. The actual thumbnail will be generated by the workflow.

POST /admin-ng/editor.json needs to update the preview because cutting the video will result in another default thumbnail which the user should see if he presses "Save".

Agree that the extraction is too slow in the community version and should not happen, though.

Reply:
@mliradelc I'll have a quick look at this now to see whether I can come up with a solution - at the end of the day, it was us that caused the problem.